src/v/arch/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+v_cc_library(
+  NAME arch
+  SRCS
+    manifest.cc
+    error.cc
+  DEPS
+    Seastar::seastar
+    v::bytes
+    v::http
+    v::s3
```

Comment:
what is `arch`

Reply:
I decided to go with this name for 'archival storage' simply because I didn't have a better one. I'll appreciate any suggestions really.

Reply:
i confuse `arch` with `architecture`. I think `archival` or `backup` would be great tho.